R/evaluate.R

```diff
@@ -0,0 +1,28 @@
+#' @title
+#' Evaluate
+#'
+#' @description
+#' Compute different metrics for discovery evaluation
+#'
+#' @param ground_truth List of true (CDR/EM) values.
+#' @param prediction List of predicted (CDR/EM) values.
+#'
+#' @keywords internal
+#'
+#' @return
+#' Intersection Over Union, Precision, Recall
+#'
+evaluate <- function(ground_truth, prediction){
+  intersect = intersect(prediction, ground_truth)
```

Comment:
Please follow the styling guide. All assignments should be implemented by `<-` rather than `=`